builder.sh

```diff
@@ -292,13 +293,19 @@ function run_build() {
         bashio::exit.nok "Invalid base image ${build_from}"
     fi
 
+    # Arch specific Dockerfile
+    if bashio::fs.file_exists "${build_dir}/Dockerfile.${build_arch}"; then
```

Comment:
VSCode uses `xxx.dockerfile` for syntax highlight, maybe use that?
eg. `amd64.dockerfile`

Reply:
Both work it seems? https://github.com/microsoft/vscode-docker/blob/main/package.json#L1510-L1513

I actually prefer the postfix one, I feel I have seen this already in other places.

Reply:
Oh cool 👍 it did not do that last I checked